crates/router/src/routes/dummy_connector/types.rs

```diff
@@ -7,15 +8,39 @@ use super::errors::DummyConnectorErrors;
 use crate::services;
 
 #[derive(Debug, Display, Clone, PartialEq, Eq)]
+#[allow(clippy::enum_variant_names)]
 pub enum Flow {
     DummyPaymentCreate,
+    DummyPaymentRetrieve,
+    DummyRefundCreate,
+    DummyRefundRetrieve,
 }
 
 impl FlowMetric for Flow {}
 
+#[allow(dead_code)]
+#[derive(Default)]
+pub enum DummyConnectorStatus {
+    Succeeded,
+    #[default]
+    Processing,
+    Failed,
+}
+
+impl std::fmt::Display for DummyConnectorStatus {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Succeeded => write!(f, "succeeded"),
+            Self::Processing => write!(f, "processing"),
+            Self::Failed => write!(f, "failed"),
+        }
+    }
+}
```

Comment:
Why are we not using `Serialize` and/or `strum::Display` here?

Reply:
Before the names were different which are changed in this PR. So now I can use strum::Display.